v2/car.go

```diff
@@ -36,17 +42,100 @@ type (
 	}
 	// Characteristics is a bitfield placeholder for capturing the characteristics of a CAR v2 such as order and determinism.
 	Characteristics struct {
+		io.WriterTo
```

Comment:
this doesn't seem like a property of the characteristics, but rather a method as you have on line 52 below

Reply:
Though not required, the idea both here and in other places, like [`Header`](https://github.com/ipld/go-car/blob/6ba30fb61f4d815422602659975e28198284e773/v2/car.go#L33) was to document that the interface implemented by the structs purely for human readability. 

This does however add an anonymous field of type `io.Writer` to the structs.
Thank you for the feedback. I will remove it now that it seem to cause confusion and therefore not serving its intended purpose.